lib/utils/randomInteger.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
This file is a copy of `test/utils/randomInteger.js`. Not sure if it is preferable to keep this file and delete the test one or keep them both to keep `lib` dependencies separate from `test` dependencies.